tests/test_model.py

```diff
@@ -383,7 +383,7 @@ def test_loss_fn():
                                  eps=test_cfg.optimizer.eps,
                                  weight_decay=test_cfg.optimizer.weight_decay)
 
-    for i in range(25):
+    for i in range(15):
```

Comment:
should I be worried about this?

Reply:
I dont think so

I think this test is super aggressive anyway
(I personally think it should be like 1 or 2 steps and we should add a check for checking param.grad being allclose, but this is fine, I guess)

25 steps of SGD with a ton of numerical diffs... Why is this how we test the loss fn???
+ the if the torch2 crossentropy is implemented diff, it'll have diff numerical diff properties